src/backend/expungeservice/record_summarizer.py

```diff
@@ -86,3 +91,15 @@ def get_label(charge: Charge):
             ]
             eligible_charges_by_date[label] = charges_tuples
         return eligible_charges_by_date
+
+    @staticmethod
+    def _build_eligible_nonconvictions_only(charges_eligible_now):
+        return charges_eligible_now and next(
+            filter(
+                lambda charge: charge.expungement_result.charge_eligibility.status
+                == ChargeEligibilityStatus.ELIGIBLE_NOW
```

Comment:
`charges_eligible_now`: once this becomes true you do not need to check for `ELIGIBLE_NOW`

Reply:
Woops, that variable name was inaccurate and based on earlier attempted logic. Now it's just all charges on the record.